### PR TITLE
Create icantthinkofanameritenow

### PR DESCRIPTION
**Pull request by icantthinkofanameritenow**
and adding changelog for my last change.

### CODE REVIEW

**Comment by Shadowmech88** [comment]
Your changelog doesn't have any changes listed in it.

**Comment by icantthinkofanameritenow** [comment]
Let's try this again.

File: html/changelogs/icantthinkofanameritenow [hunk 1]
```diff
@@ -0,0 +1,4 @@
+author: Icantthinkofanameritenow
+
+changes: 
+-tweak: Replaces Advanced Energy Gun in protolathe with a modkit that can be used on an energy gun to convert it into an AEG.
```
Comment (d3athrow) [inline html/changelogs/icantthinkofanameritenow]: need two spaces before the -

Comment (clusterfack) [inline html/changelogs/icantthinkofanameritenow]: Actually fun fact, despite people having spread that rumor for ages, it doesn't need 2 spaces before the -.

**Comment by 9600bauds** [comment]
I just copypaste the example.yml and change the author and changelog entry

**Pull request merged**